### DISCUSSION TITLE
USB to UART PICO

### DISCUSSION THREAD

**Discussion by patfrench** [hathach/tinyusb#2507] [Q&A]
Hello,

I'm working on a fairly simple project: my development has 4 UARTs, each UART must be connected to its CDC and each CDC must be webusb compatible.

I'm just starting out. Does anyone have a simple example of a USB UART bridge? For my tests I have a PICO rp2040 card.

Thanks in advance.

**Comment by donbommelon** [comment]
What I have found in my past: 
[Noltari](https://github.com/Noltari/pico-uart-bridge)
[Teensy](https://github.com/newbrain/TeensyBridge)
